doc/integrator/create_application.rst.tmpl

```diff
@@ -113,8 +113,45 @@ Go to your new project:
 
 .. prompt:: bash
 
-    cd ${GEOMAPFISH_PROJECT}
+    cd ${DOLLAR}{GEOMAPFISH_PROJECT}
 
+Simple application
+------------------
+
+A simple application is an application that you can't add come code on it (Python or JavaScript).
+
+To create a simple application you should:
+
+- Remove the geoportal job in the `.github/workflows/main.yaml` file.
+- Build the application by adding `--config` to the `./build` command.
+- Add the immage to the `geoportal` service in the `docker-compose.yaml`:
+  `image: camptocamp/geomapfish-geoportal:${MAIN_VERSION}`.
+- Add the following lines in the `.gitignore` file:
+
+.. code::
+
+    # Simple mode
+    /geoportal/*
+    !/geoportal/vars*.yaml
+    !/geoportal/CONST_vars.yaml
+    !/geoportal/CONST_config-schema.yaml
+    !/geoportal/geomapfish_geoportal
+    /geoportal/geomapfish_geoportal/*
+    !/geoportal/geomapfish_geoportal/locale/
+    !/geoportal/geomapfish_geoportal/static/
+
+    /CONST_create_template/geoportal/*
+    !/CONST_create_template/geoportal/vars*.yaml
+    !/CONST_create_template/geoportal/geomapfish_geoportal
+    /CONST_create_template/geoportal/geomapfish_geoportal/*
+    !/CONST_create_template/geoportal/geomapfish_geoportal/locale/
+    !/CONST_create_template/geoportal/geomapfish_geoportal/static/
+
+
+.. note::
+
+    If you apply thoes stepy on an application that already be committed on Git you should also remove
```

Comment:
If you apply those steps on a project already committed on Git, you should also remove